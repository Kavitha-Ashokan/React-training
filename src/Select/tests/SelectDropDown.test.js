import React from "react";
import renderer from "react-test-renderer";
import { Form } from "react-final-form";
import SelectDropDown from "../SelectDropDown";

it("it should render and match the snapshot", () => {
  const tree = renderer
    .create(
      <Form onSubmit={jest.fn()}>
        <SelectDropDown
          name="name"
          defaultValue="india"
          options={[
            { value: "India", label: "India" },
            { value: "Austraila", label: "Austraila" },
            { value: "France", label: "France" },
            { value: "Germany", label: "Germany" },
          ]}
        />
      </Form>
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
