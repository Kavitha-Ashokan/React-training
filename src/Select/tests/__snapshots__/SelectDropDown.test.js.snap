// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`it should render and match the snapshot 1`] = `null`;
